components/widgets/inputs/users_emails_input.tsx

```diff
@@ -300,6 +302,10 @@ export default class UsersEmailsInput extends React.PureComponent<Props, State>
         }
     }
 
+    componentDidMount() {
```

Comment:
I'm curious why do we need this componentDidMount?

Reply:
Oops! We absolutely don't. I was trying to reinvent `autoFocus` before I read the docs and found out it was a thing.